tests/ParserTest.php

```diff
@@ -261,6 +263,24 @@ public function testInclusion() : void
         self::assertContains('I was actually included', $document->renderDocument());
     }
 
+    public function testThrowExceptionOnInvalidFileInclude() : void
+    {
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage('Include ".. include:: non-existent-file.rst" does not exist or is not readable.');
+
+        $parser      = new Parser();
+        $environment = $parser->getEnvironment();
+
+        $environment->getConfiguration()
+            ->throwExceptionOnInvalidFileInclude(true);
+
+        $data = file_get_contents(__DIR__ . '/files/inclusion-bad.rst');
+
+        assert($data !== false);
```

Comment:
Please use the framework's test assertions: `self::assertInternalType('string', $data)` should work.